Examples/FilterSubmissions/Program.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using IndicoV2;
+using IndicoV2.Submissions.Models;
+using Newtonsoft.Json.Linq;
+
+namespace Examples
+{
+    /// <summary>
+    /// Filters Submissions for all COMPLETE or FAILED submissions that have
+    /// not been retrieved yet.
+    /// </summary>
+    public class FilterSubmissions
+    {
+        private static string GetToken() =>
+            File.ReadAllText(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile),
+                "indico_api_token.txt"));
+
+        public static async Task GetCompleteSubmissions(dynamic submissionClient)
+        {
+            // Filters for just submissions with a COMPLETE status
```

Comment:
Awesome! When the readme documentation is updated we should also link to it from this example so people can learn more about it.